index.js

```diff
@@ -1,2 +1,4 @@
+import 'react-native-gesture-handler'
```

Comment:
still needed ?

Reply:
According to the RNGH docs, this should be the first import in `index.js`: [https://docs.swmansion.com/react-native-gesture-handler/docs/installation#ios](https://docs.swmansion.com/react-native-gesture-handler/docs/installation#ios)